src/splitInteger.test.js

```diff
@@ -4,18 +4,22 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(6, 2))
+    .toEqual([3, 3]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(2, 1))
+    .toEqual([2]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(7, 3))
+    .toEqual([2, 2, 3]);
 });
 
 test('should add zeros if value < numberOfParts', () => {
```

Comment:
Update the test description to better reflect the function requirements. The function should only accept positive integers, so you don't need to test for zeros or negative values 🚀